conf/lsf.conf

```diff
@@ -39,11 +37,11 @@ backend {
             -q ${lsf_queue} \
             -n ${cpu} \
             ${"-g " + lsf_job_group} \
-            -R "select[singularity] rusage[mem=${memory_mb}] span[hosts=${hosts}]" \
+            -R "rusage[mem=${memory_mb}] span[hosts=${hosts}]" \
```

Comment:
We may want to consider adjusting the memory parameter. Generally, we specify total memory in our tasks, but in our LSF environment, the memory is per core. So if a task specifies C cores and N GB of RAM, LSF will interpret that as C * N GB of memory.

Reply:
Agreed, changed this line in `lsf.conf` to 
`-R "rusage[mem=${round(round(memory_mb)/cpu)}] span[hosts=${hosts}]"`
which should handle the memory per core.